### PR TITLE
Add version check, currently for 1.8.5

### PR DESCRIPTION
**Pull request by martinhoefling**
Version check, can be used by other features to ensure minimum gopass version

### CODE REVIEW

File: web-extension/generic.js [hunk 1]
```diff
@@ -10,6 +10,32 @@ const DEFAULT_SETTINGS = {
...
+            response.major > major ||
+            (response.major === major &&
+                (response.minor > minor || (response.minor === minor && response.patch >= patch)))
+        ) {
+            versionOK = true;
+            console.log('Version is OK');
+            return Promise.resolve();
+        }
+        console.log('Version is not OK');
+        return Promise.reject(new Error(`minimum gopass version is ${REQUIRED_GOPASS_VERSION.join('.')}`));
```
Comment (Pharb) [inline web-extension/generic.js]: @martinhoefling I would prefer something like `Please update gopass to version X or newer.`. Would that be ok for you?

Comment (martinhoefling) [reply web-extension/generic.js]: Sure, that's a way friendlier check. Do you wanna make the change?

Comment (Pharb) [reply web-extension/generic.js]: yes, I already pushed it 😉

**Comment by Pharb** [comment]
@martinhoefling I'm not sure why the codecov check is still pending, but you can merge it, as the coverage also looks fine on the codecov website. (Github does not allow me to merge the PR without that check.)

**Comment by martinhoefling** [comment]
OK, will do so.

**Pull request merged**